packages/@react-aria/dnd/src/useDropIndicator.ts

```diff
@@ -23,7 +23,9 @@ import {useLocalizedStringFormatter} from '@react-aria/i18n';
 
 export interface DropIndicatorProps {
   /** The drop target that the drop indicator represents. */
-  target: DropTarget
+  target: DropTarget,
+  /** A delegate object that implements behavior for keyboard focus movement. */
+  keyboardDelegate?: KeyboardDelegate<unknown>
```

Comment:
Update to useDropIndicator api, see discussion [here](https://github.com/adobe/react-spectrum/pull/4008#discussion_r1137904263) for background.